lib/processTextStyles.js

```diff
@@ -40,13 +40,30 @@ async function processTextStyles(textStyles, outputFolder) {
         addOptionalProperty("blur", data.style, textBlur(allStyles))
         addOptionalProperty("underline", data.text, textUnderline(encodedAttributes))
         addOptionalProperty("strikethrough", data.text, textStrikethrough(encodedAttributes))
-        ret.push(data)
+        swiftUIStyles.push(data)
+
+        // Get properties we want to make accessible for UIKit
+        const propertyData = {
+          name: style.name,
+          properties: [
+            { name: "textColor", value: `UIColor(red: ${color.red}, green: ${color.green}, blue: ${color.blue}, alpha: ${color.alpha})`},
```

Comment:
can these properties move into functions, instead of being explicitly declared here?